engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -822,3 +842,12 @@ definitions:
       jobs:
         type: integer
         description: The number of currently running jobs
+  VersionResponse:
+    description: 'Provides the version of the engine running inside the system'
```

Comment:
I think the outer quotes are superfluous?  Also maybe "Returns the version of Cromwell"